rs/tests/src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub mod networking;
 pub mod nns_dapp;
 pub mod nns_tests;
 pub mod orchestrator;
+pub mod qualification;
```

Comment:
Let's be consistent 

```suggestion
pub mod guestos_qualification;
```

Reply:
Hm, but this is the whole qualification `utils`. Maybe it will be used for more than just `questos_qualification`? I could maybe renamte it to `os_qualification_utils`

Reply:
I see. Can we rename it to ic_os_qualification_utils`?

Reply:
(Sorry for being pedantic; we regularly spend time renaming things for consistency as the lack of naming conventions across teams makes it hard to navigate in a mono repo. The least we can do is not to introduce new inconsistencies when they are trivial to overcome).

Reply:
No worries! Thanks for helping me with naming, i'm pretty terrible at it!